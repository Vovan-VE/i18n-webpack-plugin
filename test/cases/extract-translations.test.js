import processFile from '../cases.setup';

describe('extract-translations', () => {
  let extracted;

  beforeAll(() => processFile('extract-translations.code.js')
      .then(({ file }) => {
        extracted = require.requireActual(file);
      }));

  it('should return extracted keys', () => {
    expect(extracted).toMatchSnapshot();
  });
});
