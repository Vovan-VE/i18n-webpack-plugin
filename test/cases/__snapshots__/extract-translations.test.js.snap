// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`extract-translations should return extracted keys 1`] = `
Object {
  "category1": Object {
    "Message 1": "",
    "Message 2": "",
  },
  "category2": Object {
    "Message 3": "",
    "Message imported": "",
  },
}
`;
